inputs/autocomplete-item.vue

```diff
@@ -29,18 +39,30 @@
     v-bind:class="{ 'selected': isActive }"
     type="button"
     @click.stop.prevent="select(value)">
-    {{ value }}
+    <span class="item-value">{{value}}</span>
+    <span class="item-actions">
+      <ui-icon-button v-show="canDestroy" icon="clear" :tooltip="`Remove ${value} from list`" @click.stop="destroy(value)" color="default" size="small" type="secondary"></ui-icon-button>
```

Comment:
This is a little nitpicky, but the convention we've been using is putting the event handlers at (or near) the end of attributes. 😅